pkg/ai/aliclouddashscope.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2023 The K8sGPT Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ai
+
+import (
+	"context"
+	"errors"
+
+	//Due to the temporary absence of a Golang client, the OpenAI client is being used for compatibility.
+	"github.com/sashabaranov/go-openai"
+)
+
+const alicloudAIClientName = "alicloud"
+
+type DashScopeClient struct {
+	nopCloser
+
+	client     *openai.Client
+	model      string
+	temprature float32
+	topP       float32
+	topK       int32
+}
+
+func (c *DashScopeClient) Configure(config IAIConfig) error {
+	const defaultBaseURL = "https://dashscope.aliyuncs.com/compatible-mode/v1"
```

Comment:
I suggest using a custom base url for the openAI provider rather than writing a new one